csharp_dotnetcore/Using-Cards/Using_Cards/BotBuilder.ruleset

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Rules for StyleCop.Analyzers" Description="Code analysis rules for StyleCop.Analyzers.csproj." ToolsVersion="14.0">
+  <Rules AnalyzerId="AsyncUsageAnalyzers" RuleNamespace="AsyncUsageAnalyzers">
+    <Rule Id="UseConfigureAwait" Action="Warning" />
```

Comment:
I'm seeing this file in 2 directories:
Using_Cards
Using-Cards

I suspect this file should only be in there once.